app/decorators/session_decorator.rb

```diff
@@ -31,6 +31,10 @@ def cancel_link_url
     view_context.root_url
   end
 
+  def mfa_expiration_interval
+    Figaro.env.remember_device_expiration_hours_aal_1.to_i.hours
```

Comment:
If this were accidentally unset, what would the impact of this coming out to 0 hours be? (`nil.to_i` is `0`)

Reply:
What do you mean by accidentally unset? These have been added to the params require by Figaro which means if it isn't present it raises and the app crashes before starting.

Reply:
🤷‍♂️  Like if a separate bug were to cause it to be changed at runtime after application start or something. Just thinking through whether any unexpected failure conditions would manifest starkly or gracefully. If you don't think it's a big deal, don't worry about it.

Reply:
Yeah, what would happen is remember device would be completely broken, meaning people would have to MFA every time.

We're pretty careful about the Figaro env since we depend on it being stable in more places than this. For example, once these values are set on `Figaro.env` they are read-only:

```ruby
irb(main):002:0> Figaro.env.remember_device_expiration_hours_aal_2 = nil
Traceback (most recent call last):
        1: from (irb):2
NoMethodError (undefined method `remember_device_expiration_hours_aal_2=' for Figaro::ENV:Module)
```

Reply:
Very nice. Thanks for explaining that.